File: Python/src/Problem005.py
__author__ = 'Joel'

def lowest_common_multiple(a, b):
    val = a
    while val % b != 0:
        val += a
    return val

lcm = 1
for i in range(2, 20):
    lcm = lowest_common_multiple(lcm, i)
print(lcm)

File: Java/src/solutions/Problem005.java
package solutions;

import framework.Problem;
import framework.ProblemInfo;
import utilities.NumberTheory;

@ProblemInfo(
        description = "Smallest multiple",
        number = 5
)
public class Problem005 extends Problem {
    @Override
    public Object solve() {
        int lcm = 1;
        for (int i = 2; i <= 20; i++) {
            lcm = NumberTheory.lowestCommonMultiple(lcm, i);
        }
        return lcm;
    }
}